x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/connector_add_flyout.tsx

```diff
@@ -72,10 +72,8 @@ export const ConnectorAddFlyout = ({
   const [isSaving, setIsSaving] = useState<boolean>(false);
 
   const closeFlyout = useCallback(() => {
```

Comment:
Removed the following lines because I was seeing another console error when saving a connector. With this PR https://github.com/elastic/kibana/pull/82126, the connector flyouts are destroyed when closed instead of hidden, so setting these states on closing was causing a `Warning: Can't perform a React state update on an unmounted component.` browser error.